### PR TITLE
ActionItems visibility

### PR DESCRIPTION
**Pull request by kvelikov**
related to #691

### CODE REVIEW

**Comment by kvelikov** [comment]
run ci

**Comment by kvelikov** [comment]
run ci

**Comment by kvelikov** [comment]
run ci

**Comment by dtopuzov** [comment]
:+1:

**Comment by vakrilov** [comment]
:+1:

File: ui/action-bar/action-bar-common.ts [hunk 1]
```diff
@@ -238,16 +249,15 @@ export class ActionItem extends bindable.Bindable implements dts.ActionItem {
...
         "icon", "ActionItem", new dependencyObservable.PropertyMetadata(null, null, ActionItem.onItemChanged));
 
-    private static onItemChanged(data: dependencyObservable.PropertyChangeData) {
-        var menuItem = <ActionItem>data.object;
-        if (menuItem.actionBar) {
-            menuItem.actionBar.update();
-        }
-    }
+    public static visibilityProperty = new dependencyObservable.Property(
+        "visibility", "ActionItemBase", new dependencyObservable.PropertyMetadata(enums.Visibility.visible, null, ActionItem.onItemChanged));
```
Comment (hshristov) [inline ui/action-bar/action-bar-common.ts]: There is no ActionItemBase anymore

File: ui/action-bar/action-bar.android.ts [hunk 2]
```diff
@@ -208,7 +207,7 @@ export class ActionBar extends common.ActionBar {
 
     public _addActionItems() {
         var menu = this._toolbar.getMenu();
-        var items = this.actionItems.getItems();
```
Comment (hshristov) [inline ui/action-bar/action-bar.android.ts]: For android there is visibility property on MenuItem. Why don't you use it?

**Pull request merged**